index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
```

Comment:
I suggest that next time as you make your PR, please check for the Linter errors to make the code clean of errors.